### DISCUSSION TITLE
Poor docs in regarding to running the app

### DISCUSSION THREAD

**Discussion by abelncm** [CollaboraOnline/online#5717] [General]
How can I access the editor?
Only an OK message is show when opening localhost:9980

**Comment by pedropintosilva** [comment]
Hi @abelncm and thanks for the feedback, perhaps we could have this better explained and also more visible. But let me try to help here:

You need something else that would manage your storage as stated in https://sdk.collaboraonline.com/docs/faq.html?highlight=browser#do-the-programs-run-in-a-browser-and-or-as-local-clients but you can always access:
* admin console `/browser/dist/admin/admin.html` as stated in https://sdk.collaboraonline.com/docs/installation/CODE_Docker_image.html?highlight=browser#id5
* And for testing purposes you can always use the test file located in `test/data/hello-world.odt` https://github.com/CollaboraOnline/online/tree/master/test/data so if you have it running you would just append the file path like so `[...]:9980/browser/dist/debug.html?file_path=/yourfilestructure/test/data/hello-world.odt`

There are also a couple of minimal examples on how to integrate Collabora Online available as a starting point : https://github.com/CollaboraOnline/collabora-online-sdk-examples
